Policies/Kubernetes/flexvolume-drivers/azurepolicy.json

```diff
@@ -0,0 +1,59 @@
+{
+  "properties": {
+    "displayName": "Ensure only allowed FlexVolume Drivers are used in Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy ensures only allowe FlexVolume Drivers are used inside a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc",
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        },
+        "allowedValues": [
+          "enforceOPAConstraint",
+          "disabled"
+        ],
+        "defaultValue": "enforceOPAConstraint"
+      },
+      "excludedNamespaces": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Namespace exclusions",
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+        },
+        "defaultValue": []
+      },
+      "allowedFlexVolumes": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Allowed FlexVolume Drivers",
+          "description": "List of FlexVolume Drivers that volumes are able to use"
```

Comment:
Can you please add text "An empty list means there is no restriction on flexVolume drivers used" to the description?